app/cdap/components/RecipeList/RecipesTable.tsx

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React from 'react';
+import T from 'i18n-react';
+import { RecipeTableRow } from './RecipeTableRow';
+import { IRecipe } from './types';
+import EmptyMessageContainer from 'components/EmptyMessageContainer';
+import SortableHeader from './SortableHeader';
+import { IState, setSort } from './reducer';
+
+interface IRecipeTableProps {
+  allRecipies: IRecipe[];
+  isShowAllColumns: boolean;
+  isShowActions: boolean;
+  enableSorting: boolean;
+  viewHandler: (selectedRecipe: IRecipe) => void;
+  editHandler: (selectedRecipe: IRecipe) => void;
+  selectHandler?: (selectedRecipe: IRecipe) => void;
+  state: IState;
+  dispatch: (action: any) => void;
+}
+
+const PREFIX = 'features.WranglerNewUI.Recipe';
+
+export const RecipesTable = ({
+  allRecipies,
+  isShowAllColumns,
+  isShowActions,
+  enableSorting,
+  viewHandler,
+  editHandler,
+  selectHandler,
+  state,
+  dispatch,
+}: IRecipeTableProps) => {
+  const renderSortableHeaderColumn = (columnName: string) => {
```

Comment:
I'm not sure using one `sortOrder` makes sense. I feel like it you have `sortNameOrder` and `sortUpdatedOrder`. @seanfeld wdyt

Reply:
{sortColumn, sortedOrder } together tells column & order for sorting on the table. This way, we can extend sorting functionality for columns as needed in future.  I think using sortNameOrder and sortUpdatedOrder would be too specific. Will discuss with you to understand more on this.